lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
Interesting... so the kernel major.minor won't change in Win 10 Anniversary, just the last string (build number?) My inclination is to punt and not worry about additional differentiation... perhaps with a solution that would allow us to also address if/how we want to handle the incongruity of server versions strings ie "2008 R2" currently returned by this fact vs kernel version strings on consumer versions. Open to other thoughts too.